sw/device/tests/autogen/meson.build

```diff
@@ -0,0 +1,54 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+#
+# ------------------- W A R N I N G: A U T O - G E N E R A T E D   C O D E !! -------------------#
+# PLEASE DO NOT HAND-EDIT THIS FILE. IT HAS BEEN AUTO-GENERATED WITH THE FOLLOWING COMMAND:
+# util/topgen.py -t hw/top_earlgrey/data/top_earlgrey.hjson \
+# -o hw/top_earlgrey
+
+###############################################################################
+# IP Integration Tests
+###############################################################################
+plic_all_irqs_test_lib = declare_dependency(
+  link_with: static_library(
+    'plic_all_irqs_test_lib',
+    sources: ['plic_all_irqs_test.c'],
+    dependencies: [
+                  # sw_lib_dif_adc_ctrl,
```

Comment:
Indenting Mako directives with inline text makes the indentation in the generated text look horrible. It works for C because `clang-format` but not for other sources. I think we should stick to left-aligned mako code in templates.